releasenotes/notes/Add-qubit-channel-mapping-to-backend-config-schema-7f8f93034074b328.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    Added ``channels`` to the Pulse enabled backend configuration schema.
+    The ``channels`` is a dictionary where each entry represents a channel configuration
+    and contains configuration values such as the channel's mapping to qubits.
```

Comment:
Actually can we remove this release note, it's not really a user facing feature based on the review discussion.

Reply:
ok. I'll remove it. Thanks!